implementations/rust/ockam/ockam_command/src/run/parser/resources.rs

```diff
@@ -183,11 +197,10 @@ pub fn as_keyword_arg(k: &ArgKey) -> String {
 pub fn resolve(v: &ArgValue) -> String {
     match v {
         ArgValue::String(v) => {
-            if v.starts_with('$') {
-                let v = v.trim_start_matches('$');
-                if let Ok(v) = std::env::var(v) {
-                    return v;
-                }
+            if v.contains('$') {
+                return shellexpand::env(v)
+                    .expect("Failed to resolve environment variables")
+                    .to_string();
```

Comment:
Is it ok to panic w/ expect here, or are we supposed to return an error that will be wrapped into some other miette error and reported to the user (bubbling up via `run_config()`)?

Reply:
I'd keep it like this for now. This will panic if the env vars are unknown.

I'll improve this in a separate PR.

Reply:
Ok sounds good 👍🏽